src/app.scss

```diff
@@ -158,3 +158,12 @@ input {
 input:focus {
     outline: 1px solid var(--accent-color);
 }
+
+.modal {
```

Comment:
This should be moved to the Dialogue component

Reply:
Got it 👍

Reply:
> This should be moved to the Dialogue component

Done